src/UserManager.ts

```diff
@@ -328,12 +328,14 @@ export class UserManager {
         if (!url) {
             logger.throw(new Error("No silent_redirect_uri configured"));
         }
+        const user: User | null = await this.getUser();
```

Comment:
To avoid triggering an "user loaded" this should use `this._loadUser()` here as the only purpose of retrieving the current user is to provide the `id_token_hint`.

Reply:
good point, fixed!